src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.html

```diff
@@ -20,3 +20,25 @@
     {{ role }}{{ !isLast ? ", " : "" }}
   </span>
 </ng-template>
+
+<ng-template #warningTpl
+             let-column="column"
+             let-value="value"
+             let-row="row">
+  <div [ngClass]="getWarningClass(row)"
```

Comment:
@alfonsomthd already commented this on a PR from Waad, but in Angular putting methods in HTML templates is a risky thing. Please, let's better do this experiment: add a `console.log()` inside that `getWarningClass` and `getWarningIcon...` and let's count how many times that trace appears in the console... ;-)